images/module-dependencies.dot

```diff
@@ -0,0 +1,8 @@
+digraph {
+  Update -> Main
+  View -> Main
+  Model -> Main
+  Model -> Update
+  Model -> View
+  Flags -> Main
+}
```

Comment:
`rank` can probably add that constraint http://graphs.grevian.org/example#example-7